lighthouse-plugin-publisher-ads/computed/bid-request-time.js

```diff
@@ -36,6 +36,11 @@ class LanternBidRequestTime extends AdLanternMetric {
     const {nodeTimings} = simulationResult;
     const timeInMs = AdLanternMetric.findNetworkTiming(
```

Comment:
Nit: Since there are now multiple times being tracked, maybe rename this to bidRequestTimeMs?